toml/src/main/kotlin/org/rust/toml/crates/local/CratesLocalIndexServiceImpl.kt

```diff
@@ -314,15 +316,17 @@ private object CrateExternalizer : DataExternalizer<CargoRegistryCrate> {
     }
 }
 
-private fun crateFromJson(json: String): CargoRegistryCrateVersion {
-    data class ParsedVersion(
-        val name: String,
-        val vers: String,
-        val yanked: Boolean,
-        val features: HashMap<String, List<String>>
-    )
+data class ParsedVersion(
+    @JsonProperty("name") val name: String,
+    @JsonProperty("vers") val vers: String,
+    @JsonProperty("yanked") val yanked: Boolean,
+    @JsonProperty("features") val features: HashMap<String, List<String>>
+)
 
-    val parsedVersion = Gson().fromJson(json, ParsedVersion::class.java)
+private fun crateFromJson(json: String): CargoRegistryCrateVersion {
+    val parsedVersion = ObjectMapper()
```

Comment:
Consider using Kotlin module this way:
```Kotlin
    val parsedVersion = JsonMapper()
        .registerKotlinModule()
        .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
        .readValue<ParsedVersion>(json)
```
See `org.rustPerformanceTests.CustomRealProjectAnalysisTest.Companion#data` for example.